dev/com.ibm.ws.jdbc_fat_krb5/semeruFips140_3CustomProfile.properties

```diff
@@ -0,0 +1,26 @@
+# Allowing for JDBC Kerberos
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.desc.name = Custom OpenJCEPlusFIPS Cryptographic Module FIPS 140-3
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.extends = RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.jce.provider.2 = sun.security.provider.Sun [+ \
+    {KeyStore, PKCS12, *, ModuleAndFullClassName:java.base/java.security.KeyStore}, \
+    {KeyStore, JKS, *, ModuleAndFullClassName:java.base/java.security.KeyStore}, \
+    {MessageDigest, SHA-1, *, ModuleAndFullClassName:java.base/java.security.KeyStore}, \
+    {MessageDigest, SHA-1, *, ModuleAndFullClassName:java.base/com.sun.crypto.provider.HmacSHA1}]
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.jce.provider.4 = com.sun.crypto.provider.SunJCE [ \
+    {AlgorithmParameters, PBES2, *, ModuleAndFullClassName:java.base/sun.security.pkcs12.PKCS12KeyStore}, \
+    {AlgorithmParameters, PBEWithHmacSHA256AndAES_256, *, ModuleAndFullClassName:java.base/sun.security.pkcs12.PKCS12KeyStore}, \
+    {SecretKeyFactory, PBEWithMD5AndDES, *, ModuleAndFullClassName:java.base/sun.security.pkcs12.PKCS12KeyStore}, \
+    {Cipher, PBEWithHmacSHA256AndAES_256, *, ModuleAndFullClassName:java.base/sun.security.pkcs12.PKCS12KeyStore}, \
+    {Cipher, AES, *, ModuleAndFullClassName:openjceplus/com.ibm.crypto.plus.provider.AESCipher}, \
+    {Cipher, AES, *, Package:sun.security.krb5.internal.crypto}, \
+    {Cipher, HmacSHA1, *, Package:sun.security.krb5.internal.crypto}, \
+    {Mac, HmacSHA1, *, Package:sun.security.krb5.internal.crypto}, \
+    {Mac, HmacPBESHA256, *, ModuleAndFullClassName:java.base/sun.security.pkcs12.PKCS12KeyStore}]
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.jce.provider.5 = com.ibm.crypto.plus.provider.OpenJCEPlus [ \
+  {SecretKeyFactory, PBKDF2WithHmacSHA512, *, FullClassName:oracle.jdbc.driver.OracleDriver}, \
+  {MessageDigest, SHA-1, *, FullClassName:oracle.jdbc.driver.OracleDriver}]
+RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.jce.provider.6 = sun.security.jgss.SunProvider [ \
+    {GssApiMechanism, 1.2.840.113554.1.2.2, *} , \
```

Comment:
Is it possible we can add a FullClassName to line 23/24?

Reply:
While anything may be possible, my testing has found that the JDK doesn't accept us specifying the ModuleAndPackage like so
RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.jce.provider.6 = sun.security.jgss.SunProvider [ \
    {GssApiMechanism, 1.2.840.113554.1.2.2, ModuleAndPackage:java.security.jgss/sun.security.jgss}]
    
There may still be a possible way, we haven't found a way to do it yet